rule/exported.go

```diff
@@ -13,38 +14,96 @@ import (
 	"github.com/mgechev/revive/lint"
 )
 
+// IgnoredWarnings store ignored warnings types
+type IgnoredWarnings struct {
+	Const    bool
+	Function bool
+	Method   bool
+	Type     bool
+	Var      bool
+}
+
+// Name returns the rule name.
+func (i *IgnoredWarnings) isDisabled(t string) bool {
+
+	switch t {
+	case "var":
+		return i.Var
+	case "const":
+		return i.Const
+	case "function":
+		return i.Function
+	case "method":
+		return i.Method
+	case "type":
+		return i.Type
+	default:
+		panic(fmt.Sprintf("Unknown ignore warning flag %s", t))
+	}
+
```

Comment:
```suggestion
```

Reply:
removed default, but add return false after switch (by default selected kind/type is not disabled so warning can be printed )